gateway/response/response.go

```diff
@@ -66,6 +66,12 @@ const (
 	InvalidSourceID = "Invalid source id"
 	// InvalidReplaySource - Invalid replay source
 	InvalidReplaySource = "Invalid replay source"
+	// InvalidDestinationID - Invalid destination id
+	InvalidDestinationID = "Invalid destination id"
+	// DestinationDisabled - Destination is disabled
+	DestinationDisabled = "Destination is disabled"
+	// NoDestinationIdInHeader - Failed to read destination id from header
+	NoDestinationIdInHeader = "Failed to read destination id from header"
```

Comment:
Go's idiomatic capitalization
```suggestion
	NoDestinationIDInHeader = "Failed to read destination id from header"
```